quickstarts/microsoft.machinelearningservices/machine-learning-create-vnet/GettingStarted.md

```diff
@@ -0,0 +1,238 @@
+# Create Azure Machine Learning Services Resources with the Deploy to Azure Button below
```

Comment:
remove the deploy/visualize buttons from this file and refer users to the readme (or just move the content to the readme